app/src/main/res/layout/activity_homework.xml

```diff
@@ -0,0 +1,127 @@
+                                android:paddingEnd="7dp"
+                                android:paddingRight="7dp"
+                                android:text="@{viewModel.homework.courseId.name}"
+                                android:textAllCaps="true"
+                                android:textSize="@dimen/text_smaller_size"
+                                app:layout_constrainedWidth="true"
+                                app:layout_constraintBottom_toBottomOf="parent"
+                                app:layout_constraintEnd_toEndOf="parent"
+                                app:layout_constraintHorizontal_bias="0.0"
+                                app:layout_constraintStart_toStartOf="parent"
+                                app:layout_constraintTop_toTopOf="parent"
+                                tools:text="Course" />
+
+                            <ImageView
+                                android:layout_width="wrap_content"
+                                android:layout_height="wrap_content"
+                                android:paddingTop="1dp"
+                                android:src="@drawable/circle_course_color"
+                                app:setcolor="@{viewModel.homework != null? Color.parseColor(viewModel.homework.courseId.color) : Color.TRANSPARENT}"
+                                app:layout_constraintStart_toEndOf="@+id/course_title"
+                                app:layout_constraintTop_toTopOf="parent" />
+
+
+                        </android.support.constraint.ConstraintLayout>
+
+                        <ImageView
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:visibility="@{ViewUtilsKt.asVisibility(HomeworkActivity.dueLabelFlagRequired(viewModel.homework))}"
+                            app:srcCompat="@drawable/baseline_outlined_flag_16dp" />
+
+                        <TextView
+                            android:layout_width="wrap_content"
+                            android:layout_height="wrap_content"
+                            android:layout_weight="0"
+                            android:text="@{viewModel.homework.dueTextAndColorId.first}"
+                            android:textColor="@{safeUnbox(viewModel.homework.dueTextAndColorId.second)}"
+                            android:textSize="@dimen/text_smaller_size"
+                            tools:text="In x Stunden fällig" />
+
+                    </LinearLayout>
+
+                    <TextView
+                        style="@style/TextAppearance.AppCompat.Medium"
+                        android:layout_width="match_parent"
+                        android:layout_height="wrap_content"
+                        android:layout_marginTop="5dp"
+                        android:gravity="start"
+                        android:text="@{viewModel.homework.title}"
```

Comment:
The title should be shown inside the toolbar for consistency.

Reply:
Fixed

Reply:
But now you show the title in both the toolbar and the content itself. Could you please remove it here? (Should be the last change to be made)

Reply:
Shall I remove it from the toolbar or from the content?

Reply:
From the content

Reply:
Fixed. But now that isn't consistent with the IOS version anymore